Samples/src/UnitTesting/CCNOTCircuits.qs

```diff
@@ -148,30 +112,23 @@ namespace Microsoft.Quantum.Samples.UnitTesting {
     /// # References
     /// - [ *Michael A. Nielsen , Isaac L. Chuang*,
     ///     Quantum Computation and Quantum Information ](http://doi.org/10.1017/CBO9780511976667)
-    operation CCNOT1 (control1 : Qubit, control2 : Qubit, target : Qubit) : Unit {
-
-        body (...) {
-            H(target);
-            CNOT(control1, target);
-            Adjoint T(target);
-            CNOT(control2, target);
-            T(target);
-            CNOT(control1, target);
-            Adjoint T(target);
-            CNOT(control2, target);
-            T(target);
-            Adjoint T(control1);
-            CNOT(control2, control1);
-            H(target);
-            Adjoint T(control1);
-            CNOT(control2, control1);
-            T(control2);
-            S(control1);
-        }
-
-        adjoint self;
```

Comment:
self should be preserved for optimization purposes (it's tracked as inferred property in the compiler)

Reply:
Reverted.